File: Hello.py
#printing hello world 

print ("hello-world")


def ask_age():
    try:
        age = float(input("Enter your age: "))

        if age <= 0:
            print("Please enter a valid age!")
            ask_age()  # ask again

        elif age >= 18:
            print("You can drive")
        else:
            print("Underage! You cannot drive")

    except ValueError:
        print("Please enter a valid number!")
        ask_age()  # ask again

# Call the function
ask_age()

File: Hello.go
package main
import ("fmt" 
"math")

// created a function to ask age from user 

func askage()  {

	age:= 0.0

	fmt.Println("enter your age : ")
    fmt.Scan(&age)
     
    if math.IsNaN(age) || age <= 0 {

		println("Please enter a valid age! ")

		askage()

	}else if age >= 18 {
		println("You can drive")
	}else {
		println("underage! you cannot drive")
	}

	
}

func main(){
	fmt.Println("hello_world")

// we can declare a variable with two methods

//first method to declare a vraiable 
//var age int  

// second method to declare variable

//age:= 0


askage() // calling the function in main 
	
}
